frontend/pages/contact.js

```diff
@@ -66,43 +66,88 @@ const Contact = ({ joinUs, connectUs }) => {
             <main className="flex flex-col gap-12 py-8">
                 <div className="flex flex-col gap-1">
                     <h2 className="text-xl font-bold">Join our organisation!</h2>
-                    <div className="sm:flex sm:justify-center lg:justify-evenly">
-                        {
-                            joinUs.map(({ url, name }, i) => (
-                                <div className="mt-3 sm:mt-0 sm:ml-3" key={i}>
-                                    <Link href={url} target="_blank" passHref>
-                                        <Button className={styles.contactSocialMedia} role="button">
-                                            {name}
-                                        </Button>
-                                    </Link>
-                                </div>
-                            ))
-                        }
+                    <div className="sm:w-full flex flex-wrap justify-evenly lg:justify-evenly">
+                        <div className="mt-3 sm:mt-0 sm:ml-3">
+                            <Link href="https://forms.gle/go6DKruGaZUyQwLp9" target="_blank" passHref>
+                                <Button className={styles.contactSocialMedia} role="button">
+                                    Tutee Sign-Up
+                                </Button>
+                            </Link>
+                        </div>
+                        <div className="mt-3 sm:mt-0 sm:ml-3">
+                            <Link href="https://forms.gle/rGfEasoyakNT4oMb8" target="_blank" passHref>
+                                <Button className={styles.contactSocialMedia} role="button">
+                                    Tutor Sign-Up
+                                </Button>
+                            </Link>
+                        </div>
+                        <div className="mt-3 sm:mt-0 sm:ml-3">
+                            <Link href="https://forms.gle/1URf8q3MxHeFzefF9" target="_blank" passHref>
+                                <Button className={styles.contactSocialMedia} role="button">
+                                    EXCO Sign-Up
+                                </Button>
+                            </Link>
+                        </div>
```

Comment:
Sorry, why was this change made?

Reply:
Hihi felix, urr i think i didnt make a pull request before i did more edits, and then just pushed to the branch ;-;
Really sry abt it! But will keep note abt it in future....